.github/workflows/specimin_evaluation_CI.yml

```diff
@@ -0,0 +1,204 @@
+name: specimin_evaluation_CI 
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  specimin-evaluation:
+    runs-on: ubuntu-latest
+
+    steps:
+        
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+
+      - name: Debug - Show Current Branch
+        run: |
+          set -ex
+          echo "Current branch: $(git branch --show-current)"
```

Comment:
This step (and other debug steps) should be removed before we merge this script.